lights.json

```diff
@@ -1,84 +1,117 @@
+        "description": "A basic Lightning strike, randomly flashes on for a short time",
+        "intensityDescription": "Controls how bright the flash will be",
+        "speedDescription": "Controls how often the lightning will strike"
+      },
+      {
+        "translationName": "electricFault",
+        "name": "Electric Fault",
+        "animationFunction": "blitzElectricFault",
+        "description": "A glitched electrical light. Dims slightly at random",
+        "intensityDescription": "Controls how much dimmer the light will get",
+        "speedDescription": "Controls how often the fault will occur"
+      },
+      {
+        "translationName": "flash",
+        "name": "Simple Flash",
+        "animationFunction": "blitzSimpleFlash",
+        "description": "A simple flashing light",
+        "intensityDescription": "Controls how dim the light will be when it flashes off",
+        "speedDescription": "Controls how often the light will flash"
+      },
+      {
+        "translationName": "rgbflash",
+        "name": "RGB Flash",
+        "animationFunction": "blitzRGBFlash",
+        "description": "A simple flashing light, cycling between Red, Green and Blue",
+        "intensityDescription": "Controls how dim the light will be when it flashes off",
+        "speedDescription": "Controls how often the light will flash"
+      },
+      {
+        "translationName": "policeflash",
+        "name": "Police Flash",
+        "animationFunction": "blitzPoliceFlash",
+        "description": "A simple flashing light, cycling between Red, and Blue, mimicking police lights",
+        "intensityDescription": "Controls how dim the light will be when it flashes off",
+        "speedDescription": "Controls how often the light will flash"
+      },
+      {
+        "translationName": "torch",
+        "name": "Alternate Torch",
+        "animationFunction": "blitzTorch",
+        "description": "A deeper, alternative version of the torch"
+      },
+      {
+        "translationName": "blurtorch",
+        "name": "Blurred Torch",
+        "animationFunction": "blitzTorchBlur",
+        "description": "A deeper, alternative version of the torch"
+      }
+    ]
+  },
+  "SecretFire": {
+    "discord": "SecretFire#4843",
+    "links": {
+      "github": "https://github.com/Feu-Secret",
+      "twitter": "",
+      "youtube": ""
+    },
+    "lights": [
+      {
+        "translationName": "gridForceField",
+        "name": "Grid Force-Field",
+        "animationFunction": "secretFireAnimateTime",
+        "shaders": {
+          "coloration": "CLForceGridColorationShader"
+        },
+        "description": "A pulsating grid force-field",
+        "intensityDescription": "Controls the pulsating power",
+        "speedDescription": "Controls how quickly the grid is pulsating"
+      },
+      {
+        "translationName": "smokePatch",
+        "name": "Smoke Patch",
+        "animationFunction": "secretFireAnimateTime",
```

Comment:
Since `secretFireAnimateTime` just includes `foundryTime` and does nothing else, we could just set `foundryTime` here, instead of creating a new function